<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tagsSettingPageWidget</class>
 <widget class="QWidget" name="tagsSettingPageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="filterEditWidget">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Filter (comma separated tags)</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addTagBtnWidget">
       <property name="toolTip">
        <string>Add new tag</string>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../../../../../../../Documents/Trilhas Perdidas/Geoprocessamento/icones/qgis gui/png/symbologyAdd.png</normaloff>../../../../../../../../../../Documents/Trilhas Perdidas/Geoprocessamento/icones/qgis gui/png/symbologyAdd.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delTagBtnWidget">
       <property name="toolTip">
        <string>Remove selected tag</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../../../../../../../Documents/Trilhas Perdidas/Geoprocessamento/icones/qgis gui/png/symbologyRemove.png</normaloff>../../../../../../../../../../Documents/Trilhas Perdidas/Geoprocessamento/icones/qgis gui/png/symbologyRemove.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cloneTagBtnWidget">
       <property name="toolTip">
        <string>Clone selected tag</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../../../../../../../Documents/Trilhas Perdidas/Geoprocessamento/icones/qgis gui/png/clone.png</normaloff>../../../../../../../../../../Documents/Trilhas Perdidas/Geoprocessamento/icones/qgis gui/png/clone.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="tagTreeWidget">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::NoDragDrop</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::IgnoreAction</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Tag identify mode:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="identifyStrategyComboWidget">
       <item>
        <property name="text">
         <string>Layer name starts with tag</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Category metadata contains tag</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
